PrismSampleApp/PrismSampleApp/Model/WebApi.cs

```diff
@@ -0,0 +1,199 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+
+namespace PrismSampleApp.Model
+{
+    public class Name
+    {
+        [JsonProperty("title")]
+        public string Title { get; set; }
+
+        [JsonProperty("first")]
+        public string First { get; set; }
+
+        [JsonProperty("last")]
+        public string Last { get; set; }
+    }
+
+    public class Street
+    {
+        [JsonProperty("number")]
+        public int Number { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+    }
+
+    public class Coordinates
+    {
+        [JsonProperty("latitude")]
+        public string Latitude { get; set; }
+
+        [JsonProperty("longitude")]
+        public string Longitude { get; set; }
+    }
+
+    public class Timezone
+    {
+        [JsonProperty("offset")]
+        public string Offset { get; set; }
+
+        [JsonProperty("description")]
+        public string Description { get; set; }
+    }
+
+    public class Location
+    {
+        [JsonProperty("street")]
+        public Street Street { get; set; }
+
+        [JsonProperty("city")]
+        public string City { get; set; }
+
+        [JsonProperty("state")]
+        public string State { get; set; }
+
+        [JsonProperty("country")]
+        public string Country { get; set; }
+
+        [JsonProperty("postcode")]
+        public object Postcode { get; set; }
+
+        [JsonProperty("coordinates")]
+        public Coordinates Coordinates { get; set; }
+
+        [JsonProperty("timezone")]
+        public Timezone Timezone { get; set; }
+    }
+
+    public class Login
+    {
+        [JsonProperty("uuid")]
+        public string Uuid { get; set; }
+
+        [JsonProperty("username")]
+        public string Username { get; set; }
+
+        [JsonProperty("password")]
+        public string Password { get; set; }
+
+        [JsonProperty("salt")]
+        public string Salt { get; set; }
+
+        [JsonProperty("md5")]
+        public string Md5 { get; set; }
```

Comment:
Don't use int in naming property.

Reply:
It is a auto generated code while converting to Json code